src/plugins/mf_priority.cpp

```diff
@@ -168,15 +168,24 @@ static int get_queue_info (
 }
 
 
-static void split_string (char *queues, struct bank_info *b)
+static void split_string (char flag, char *list, struct bank_info *b)
```

Comment:
I think I see what you're saying! Adjusting the parameters of the function to just take a string to split on `,` and the vector to push the results cleans the code up quite considerably (no having to pass funky single-character flags); instead it can just look like this:

```
split_string_and_push_back (projects, &b->projects);
```

and the function only needs one `while` loop. Thanks for the suggestion!

I can make this change as well as add the callers of this function to the same commit (sorry about that!!).